.gitignore

```diff
@@ -1,7 +1,9 @@
 *.egg-info/
+.coverage
```

Comment:
Revert changes to this files, and use tox to manage your development environments.